app/models/user.rb

```diff
@@ -1,9 +1,13 @@
 class User < ApplicationRecord
+	before_save :downcase_email
 	validates :name, presence: true, length: { maximum: 50 }
 	VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
 	validates :email, presence: true, length: { maximum: 255 },
-			  format: { with: VALID_EMAIL_REGEX },
-			  uniqueness: true
+							  format: { with: VALID_EMAIL_REGEX },
+							  uniqueness: true
 	has_secure_password
 	validates :password, presence: true, length: { minimum: 6 }
-end
\ No newline at end of file
+	def downcase_email
```

Comment:
đặt trong private nhé
```suggestion
        private
	def downcase_email
```